backend-tests/tests/test_inventory.py

```diff
@@ -37,7 +40,7 @@
 )
 
 
-@pytest.yield_fixture(scope="function")
+@pytest.fixture(scope="function")
```

Comment:
this is because we bumped pytest/python itself?

Reply:
yeah, I noticed in the API docs that the `yield_fixture` is deprecated - the two decorators are the same now, but it might get removed in the future.